src/main/java/org/team1540/robot2019/subsystems/Drivetrain.java

```diff
@@ -144,90 +145,98 @@ private void configTalonsForPosition() {
         }
     }
 
-    private void configTalonsForVelocity() {
+    public void configTalonsForVelocity() {
```

Comment:
This should be made private and just called in setVelocity methods etc; there's no overhead to doing this in a loop

Reply:
Done.